server/migrations/20181023192004-create-user.js

```diff
@@ -9,7 +9,7 @@ export default {
     },
     username: {
       type: Sequelize.STRING,
```

Comment:
whats happening here?

Reply:
I had already effected this changes directly on the migration file before you gave us the feedback to always create a different migration file. I changed the null constraints because I'm not getting some user details from social media so I needed to change the constraints to save user data in the database